docker/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG PYREX_BASE=ubuntu-18.04
+ARG PYREX_BASE=ubuntu-18.04-oe
```

Comment:
You also need to change the default in `pyrex.ini` under the `config:dockerfile` key.

Reply:
you mean `config:dockerimage` right? I updated that.

Reply:
I thought I did this at this line:
https://github.com/garmin/pyrex/blob/f1c11666a0b563711bcf174793f33b951aa1af04/pyrex.ini#L38